examples/server/public/index.html

```diff
@@ -434,9 +451,12 @@
             )
         ).join("\n"),
       });
-
+      if (selected_image) {
+        prompt = `A chat between a curious human and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the human's questions.\nUSER:[img-10]${msg}\nASSISTANT:`;
```

Comment:
image slot id is statically set to 10 here. I think that might the reason for the parallel LLaVA issue.

Reply:
Ok, let's fix this later then

Reply:
The images are loaded for each slot, so it doesn't matter if it's 10, as it ultimately loads a different slot.